api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/ModelLoader.kt

```diff
@@ -708,6 +709,42 @@ private fun createConv2D(config: LayerConfig, name: String): Conv2D {
     )
 }
 
+private fun createConv3D(config: LayerConfig, name: String): Conv3D {
+    val kernelSize = config.kernel_size!!.map { it.toLong() }.toLongArray()
+    val strides = config.strides!!.map { it.toLong() }.toLongArray()
+
+    val addedOnesStrides = LongArray(5)
+    addedOnesStrides[0] = 1
```

Comment:
hmm, seems like we have no chance to serialize all 5d arrays (strides, dilations and so on) https://github.com/JetBrains/KotlinDL/discussions/110 yet one argument to reject them

Reply:
just a comment for me